0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<p>Slightly further on we say "So&nbsp;<code>vmap</code>&nbsp;should be similar in performance to manually vectorized code (if everything is implemented well), and much better than naively vectorized code (i.e. for loops)". Should we rather use the world "manually" here and "naively" above? I found myself wondering if "natively" and "naively" were supposed to be the same word and there was a typo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='118'/>

Reply:
<p>Yes, great point! Changed! </p>